lib/manifest.js

```diff
@@ -217,6 +217,15 @@ module.exports = [
             return internals.passthruOn('identity')(filename, value);
         }
     },
+    {   // Schwifty models
+        place: 'schwifty-models',
+        method: 'schwifty',
+        list: true,
+        useFilename: (filename, value) => {
+
+            return [].concat(value);
```

Comment:
Just ensures that `value` is an array.  It does look like this isn't needed, so I can simplify this quite a bit.